owncloudData/src/main/java/com/owncloud/android/data/files/db/FileDao.kt

```diff
@@ -538,6 +543,12 @@ interface FileDao {
             WHERE parentId = :folderId AND mimeType LIKE :mimeType || '%'
         """
 
+        private const val SELECT_FILES_FOR_ACCOUNT = """
+            SELECT *
+            FROM ${ProviderMeta.ProviderTableMeta.FILES_TABLE_NAME}
+            WHERE owner = :accountOwner AND keepInSync = '0'
+        """
```

Comment:
You can improve this by getting just the files that we know are available locally... Imagine an account with 1 million files and we just want to remove 2. Does it make sense to return every file? It would be a waste of resources

Reply:
You are right, done